chichiwang/exercise-js
# Layout: Add site Footer

## Description
Linked to Issue: #136 
Add a `Footer` component to `app/Layout`. The footer contains credit to the author and links to my Github profile.

In creating the footer, the rules defined in `app/Layout/styles.nav.module.scss` are consolidated in `app/Layout/styles.module.scss` and the separate stylesheet module is removed.

## Changes
* Create component `app/Layout/Footer.tsx` to be used in the Layout
* Use `app/Layout/styles.module.scss` instead of `app/Layout/styles.nav.module.scss` in `app/Layout/Nav.tsx`
* Render the new Layout Footer in `app/Layout/index.tsx`
* Add to `app/Layout/styles.module.scss`
  * Footer styles
  * NavBar styles (copied over from `app/Layout/styles.nav.module.scss`)
* Delete `app/Layout/styles.nav.module.scss`

## Steps to QA
* Pull down and switch to this branch
* Run the app and verify the footer looks good in the browser
  * Verify print styles show the divider as black
